mobile2/src/Environment.ts

```diff
@@ -3,7 +3,7 @@ import { Environment, Network, RecordSource, Store } from "relay-runtime";
 const store = new Store(new RecordSource());
 
 const network = Network.create((operation, variables) => {
-  return fetch("http://localhost:4000/graphql", {
+  return fetch("http://192.168.0.114:4000/graphql", {
```

Comment:
This local IP address shouldn't be committed as it won't work for anyone else :)

```suggestion
  return fetch("http://localhost:4000/graphql", {
```